tools/genStrSession.py

```diff
@@ -25,9 +26,14 @@ async def genStrSession() -> None:  # pylint: disable=missing-function-docstring
             api_hash=os.environ.get("API_HASH") or input("Enter Telegram API HASH: ")
     ) as userge:
         print("\nprocessing...")
-        await userge.send_message(
+        doneStr = "sent to saved messages!"
+        try:
+            await userge.send_message(
             "me", f"#USERGE #HU_STRING_SESSION\n\n```{await userge.export_session_string()}```")
-        print("Done !, session string has been sent to saved messages!")
+        except UserIsBot:
+            doneStr = "has been successfully printed!"
```

Comment:
Sup @TomyPrs 

```suggestion
            doneStr = "successfully printed!"
```

Cuz its already getting printed in line 36